tests/test_descriptive_statistics.py

```diff
@@ -1,5 +1,8 @@
 from linguaf import descriptive_statistics as ds
```

Comment:
Please, check the output of the tests within this PR. Something doesn't work as expected

Reply:
for 3/4 of the test this was related to missing language models. I've made changes to address this issue.